airflow/contrib/operators/file_to_gcs.py

```diff
@@ -49,6 +52,7 @@ def __init__(self,
                  bucket,
                  google_cloud_storage_conn_id='google_cloud_default',
                  mime_type='application/octet-stream',
+                 gzip=False,
```

Comment:
One last comment, can you move the `gzip` argument to the very end? Otherwise backward compatibility will be broken.